Formula/ghc.rb

```diff
@@ -1,10 +1,17 @@
 class Ghc < Formula
   desc "Glorious Glasgow Haskell Compilation System"
   homepage "https://haskell.org/ghc/"
-  url "https://downloads.haskell.org/~ghc/8.10.3/ghc-8.10.3-src.tar.xz"
-  sha256 "ccdc8319549028a708d7163e2967382677b1a5a379ff94d948195b5cf46eb931"
   license "BSD-3-Clause"
 
+  stable do
+    url "https://downloads.haskell.org/~ghc/8.10.3/ghc-8.10.3-src.tar.xz"
+    sha256 "ccdc8319549028a708d7163e2967382677b1a5a379ff94d948195b5cf46eb931"
+    patch do
+      url "https://gist.githubusercontent.com/hjelmn/49f70df40da0399006031d854f73473e/raw/601bed7124f0fb9787d94806eab61645b85dab0a/ghc-8.10.3.patch"
+      sha256 "5c4747103802e717ea20af05898387058b07bf0a73d38bb14c6e1ea2173baba6"
+    end
+  end
```

Comment:
There's no more `head` block/method so you can move all of this out of the `stable do` block.

Reply:
Done.